src/daft-scan/src/glob.rs

```diff
@@ -78,6 +78,10 @@ impl From<Error> for DaftError {
     }
 }
 
+fn is_interrupt_like_error(err: &DaftError) -> bool {
```

Comment:
I'm curious why `interrupt_like` here to describe the transient byte stream errors?